region/lowernorfair/east.json

```diff
@@ -3090,22 +3221,73 @@
                   "notable": false,
                   "requires": [
                     "h_canNavigateHeatRooms",
-                    {"heatFrames": 50}
+                    {"heatFrames": 40}
                   ]
                 }
               ]
             },
             {
-              "id": 3,
+              "id": 7,
               "strats": [
                 {
                   "name": "Base",
                   "notable": false,
                   "requires": [
                     "h_canNavigateHeatRooms",
-                    {"heatFrames": 550}
+                    {"or": [
+                      {"and": [
+                        "ScrewAttack",
+                        {"heatFrames": 540}
+                      ]},
+                      {"and": [
+                        "Charge",
+                        "Wave",
+                        "Ice",
+                        "Plasma",
+                        {"heatFrames": 600}
+                      ]}
+                    ]}
+                  ]
+                },
+                {
+                  "name": "Fast Descent",
+                  "notable": false,
+                  "requires": [
+                    "h_canNavigateHeatRooms",
+                    {"or": [
+                      {"and": [
+                        "canPrepareForNextRoom",
+                        "Charge"
+                      ]},
+                      "Spazer",
+                      "Plasma",
+                      "Wave"
+                    ]},
+                    {"or": [
+                      "canDownBack",
+                      "canCarefulJump"
+                    ]},
+                    {"heatFrames": 510}
                   ],
-                  "note": "The KiHunters are much easier to avoid coming in from above, no need to kill them"
+                  "note": "By breaking the shot blocks and maneuvering through the room effeciently Samus will not have to spend much time waiting on KiHunters."
+                },
+                {
+                  "name": "Wait For KiHunters",
+                  "notable": false,
+                  "requires": [
+                    "h_canNavigateHeatRooms",
+                    {"heatFrames": 1200}
+                  ]
+                },
+                {
+                  "name": "Plasma Run Through Enemies",
```

Comment:
It's a bit surprising to me that plasma + hitbox is more heat frames than plasma + careful jump. Is that correct?

Reply:
There should be a combination `Fast Descent + canHitbox` tbh.
HitBox route gets a bad cycle with top Ki where you can't just jump on top of it.  
Careful jump route can beat that cycle but has to wait a little bit on Ki 3.